src/Car.js

```diff
@@ -0,0 +1,36 @@
+import { pickNumberInRange, printResult } from './util/missionUtil.js';
+
+class Car {
+  #name;
+  #distance = '';
+  #min = 0;
+  #max = 9;
+
+  constructor(name) {
+    this.#name = name;
+  }
+
+  async getMoveForwardResult() {
+    const RANDOM_VALUE = await pickNumberInRange(this.#min, this.#max);
+    const IS_OVER_4 = RANDOM_VALUE >= 4;
+    if (IS_OVER_4) {
+      this.setDitance('-');
+    }
+    const CAR_STATUS_STRING = `${this.#name} : ${this.#distance}\n`;
+    return CAR_STATUS_STRING;
```

Comment:
위와 동일한 이유로, 변수로 새로 할당한 값을 return하기보다는 바로 return하는 것이 더 깔끔할 것 같습니다 :)

```suggestion
    return `${this.#name} : ${this.#distance}\n`;
```